source/agora/utils/PrettyPrinter.d

```diff
@@ -256,6 +256,14 @@ private struct OutputFmt
     assert(format("%s", OutputFmt(output)) == "GAAA...AWHF(0)");
 }
 
+string startNewLineIfMultiple (size_t chunk_size, size_t length) @safe nothrow
```

Comment:
It doesn't pass the rule of three test, and it's a two liner function. It can be inlined.

Reply:
Removed